src/cms/static/charCounter.js

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Please keep the handling of static files consistent with the existing files. Do not create JS files directly in the `src/cms/static` directory, but place them inside the [src/frontend/js](https://github.com/Integreat/integreat-cms/tree/develop/src/frontend/js) directory and add them to [src/frontend/index.ts](https://github.com/Integreat/integreat-cms/blob/develop/src/frontend/index.ts) to make sure they are bundled with Webpack and available on every html page.
Also, place use the file extension `ts` for TypeScript.
Please delete this file `src/cms/static/charCounter.js` from the index again - the `src/cms/static` is listed in the [.gitignore](https://github.com/Integreat/integreat-cms/blob/ef44282cb1cd16ddaa5fab7ceeefca47629adca3/.gitignore#L38) on purpose - it is meant for automatically generated static files by Webpack only.

Also see the documentation on Frontend bundling: https://integreat.github.io/integreat-cms/frontend-bundling.html